pkgs/test_core/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
   analyzer: '>=3.3.0 <5.0.0'
   async: ^2.5.0
   args: ^2.0.0
-  boolean_selector: ^2.1.0
+  boolean_selector: '>=2.1.0 <4.0.0'
```

Comment:
It looks like 2.1.0 is the latest version here, (also this isn't frontend_server_client?)

Reply:
🤦 thanks